WebDevStudios/nextjs-wordpress-starter
# Feature/25 storybook mdx

Closes https://github.com/WebDevStudios/nextjs-wordpress-starter/issues/25

### Link

https://nextjs-wordpress-starter-7oyw97kdr.vercel.app/

### Description

* Update storybook config to support .mdx stories
* Restructure component folders

### Screenshot

I refactored the alert component to use `.mdx`:
https://5fe0becf19ad53002147b034-jaybrqedhj.chromatic.com/?path=/story/design-system-molecules-alert--info

### Verification

How will a stakeholder test this?

1. Checkout the branch
1. Run `yarn storybook`
1. Verify that it works and you can write `.mdx` stories
